### PR TITLE
UI tests - Add PS version 1.7.0.6 to 1.7.8.11 on php 7.1

### PR DESCRIPTION
**Pull request by nesrineabdmouleh**
| Questions         | Answers
| ------------------| -------------------------------------------------------
| Description?      | Add PS version 1.7.0.6 to 1.7.8.11 on php 7.1
| Type?             | new feature
| Sponsor company   |
| How to test?      | CI is :green_apple:

### CODE REVIEW

**Comment by Quetzacoalt91** [comment]
The error is happening because an URL we depend on to get data is now invalid.

It has been fixed on 1.7.1.0 with https://github.com/PrestaShop/PrestaShop/pull/7597. We cannot fix this issue without modifying the core files on the fly to change the URL before starting the installation.
If these versions can't be installed anymore (1.7.0.*), they may be removed from the test matrix.

What do you think @nesrineabdmouleh & @gericfo?

File: .github/workflows/ui-test/sanity.json [hunk 1]
```diff
@@ -1,5 +1,50 @@
...
+      "PS_VERSION_END": "1.7.8.11",
+      "UPGRADE_CHANNEL": "minor",
+      "PHP_VERSION": "7.1"
+    },
+    {
+      "PS_VERSION_START": "1.7.0.5",
+      "PS_VERSION_END": "1.7.8.11",
+      "UPGRADE_CHANNEL": "minor",
+      "PHP_VERSION": "7.1"
+    },
```
Comment (Quetzacoalt91) [inline .github/workflows/ui-test/sanity.json:41]:
```suggestion
```

As you discussed with @M0rgan01, 1.7.0.6 seems to be the only 1.7.0 version with the requested fix about CLDR.

**Review by Progi1984 — DISMISSED** [review]
@nesrineabdmouleh The CI is red

**Comment by nesrineabdmouleh** [comment]
> @nesrineabdmouleh The CI is red

Yes it wip but to run the jobs the PR should not be in draft

**Review by Progi1984 — APPROVED** [review]

**Pull request merged**